src/main/java/seedu/address/logic/commands/interview/ScoreCommand.java

```diff
@@ -16,6 +16,8 @@
  */
 public class ScoreCommand extends Command {
     public static final String MESSAGE_SUCCESS = "Scored %.2f to %s";
+    private static final String MESSAGE_SCORE_OUT_OF_BOUND = "The score provided: %s is out of range. The score "
+            + "should ranges from 0 to 10, inclusive.";
```

Comment:
Did you add the check for score range? I can only see addition of message

Reply:
resolved